mobile/v1/redshift/README.md

```diff
@@ -0,0 +1,236 @@
+# Redshift v1 mobile model README
+
+This readme contains a quickstart guide, and details of how the modules interact with each other. For a guide to configuring each module, there is a README in each of the modules' `playbooks` directory.
+
+To customise the model, we recommend following the guidance found in the README in the `sql/custom` directory.
+
+## Quickstart
+
+### Prerequisites
+
+[SQL-runner](https://github.com/snowplow/sql-runner) must be installed, and a dataset of mobile events from either the Snowplow [iOS tracker](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/objective-c-tracker/) or [Android tracker](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/android-tracker/) must be available in the database. The session context and screen view events most both be enabled for the mobile model to run. 
+
+### Configuration
+
+#### Authentification
```

Comment:
Not sure whether this is personal preference or not, but I _think_ the more commonly used word is `Authentication`?

Reply:
English was never my strong point :)  Changed